go/logger/standard.go

```diff
@@ -350,3 +349,15 @@ func PickFirstError(errors ...error) error {
 func (log *Standard) SetExternalHandler(handler ExternalHandler) {
 	log.externalHandler = handler
 }
+
+type UnforwardedLogger Standard
+
+func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
+	return (*UnforwardedLogger)(log)
+}
+
+func (log *UnforwardedLogger) Debug(s string, args ...interface{})   { log.internal.Debug(s, args...) }
+func (log *UnforwardedLogger) Error(s string, args ...interface{})   { log.internal.Error(s, args...) }
```

Comment:
What I mean is, is this secretly going to call the _old_ implementation of `Error`? I don't know how virtual method calls work at this level of detail.